src/c-writer.cc

```diff
@@ -603,6 +603,7 @@ void CWriter::DefineImportName(const Import* import,
   global_syms_.insert(mangled);
   bool success =
       global_sym_map_.emplace(name + MangleField(type), mangled).second;
+  WABT_UNUSED(success);
```

Comment:
I think the idea is that it goes on the line with the declaration.. no need for this extra line.